src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1490,8 +1490,15 @@ export const ComputeModalBase = ({
             existingRuntime?.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp)
           ])],
           [willRequireDowntime(), () => h(Fragment, [
-            existingRuntime && existingRuntime.tool !== desiredTool ? p(['By continuing, you will be changing the application of your cloud environment from ', strong([existingRuntime.tool]), ' to ',
-        desiredToolPhrase, '.']) : undefined,
+            existingRuntime && existingRuntime.tool !== desiredTool ?
+              p([
+                'By continuing, you will be changing the application of your cloud environment from ',
+                strong([existingRuntime.tool]),
+                ' to ',
+                desiredToolPhrase,
```

Comment:
might be a bit more concise to condense the last three strings into one when its split up, like this 

` to ${desiredToolPhrase}.`

Reply:
Noted for next time! Would this be something I'd do with string interpolation (`\` to ${desiredToolPhrase}.\``) or does the react-hyperscript-helpers library handle interpolation its own way?

Reply:
It'd be via string interpolation (i.e. `to ${desiredToolPhrase}.` ).